.buildkite/pipelines/pull-request/bwc-snapshots-windows.yml

```diff
@@ -0,0 +1,21 @@
+config:
+  allow-labels: test-windows
+steps:
+  - group: bwc-snapshots-windows
+    steps:
+      - label: "{{matrix.BWC_VERSION}} / bwc-snapshots-windows"
+        key: "bwc-snapshots-windows"
+        command: .\.buildkite\scripts\run-script.ps1 bash .buildkite/scripts/windows-run-gradle.sh
+        env:
+          BWC_VERSION: "{{matrix.BWC_VERSION}}"
```

Comment:
I don't think this env var is actually used anywhere. We just need to pass the task name.